sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -147,14 +157,6 @@ abstract class JdbcDialect extends Serializable {
     case arrayValue: Array[Any] => arrayValue.map(compileValue).mkString(", ")
     case _ => value
   }
-
-  /**
-   * Return Some[true] iff `TRUNCATE TABLE` causes cascading default.
-   * Some[true] : TRUNCATE TABLE causes cascading.
-   * Some[false] : TRUNCATE TABLE does not cause cascading.
-   * None: The behavior of TRUNCATE TABLE is unknown (default).
-   */
-  def isCascadingTruncateTable(): Option[Boolean] = None
```

Comment:
We can't drop this, because this is a public API.

Reply:
As a middle ground, could it be marked deprecated? Seeing as we don't really need it anymore...

Reply:
Although, custom third party dialects may want it.